src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/OperationNameTelemetryInitializer.cs

```diff
@@ -40,74 +24,9 @@ protected override void OnInitializeTelemetry(HttpContext platformContext, Reque
                 {
                     // We didn't get BeforeAction notification
                     string name = platformContext.Request.Method + " " + platformContext.Request.Path.Value;
-                    requestTelemetry.Name = name;
```

Comment:
Why is this gone?  (I'm sure there's a good reason I just don't have intellisense or anything in the review to investigate).